src/color-slider/color-slider.js

```diff
@@ -99,10 +99,10 @@ const Self = class ColorSlider extends NudeElement {
 				this.style.setProperty("--color", displayedColor);
 			}
 		}
-		else if (name === "value") {
+		else if (name === "value" || name === "min" || name === "max") {
```

Comment:
This could also be a direct commit.

Reply:
Let me commit this change and the one above (with a fix for the typo) directly and rebase this branch so we can iterate.